src/__tests__/Search.test.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Search from "../../src/components/Search";
+
+describe("Search", () => {
+  test("renders correctly", () => {
+    const { asFragment } = render(<Search />);
+
+    expect(asFragment()).toMatchSnapshot();
+    
+  });
+})
```

Comment:
I would recommend adding more tests and not just relying solely on snapshot tests. e.g. you could test:
- that the search field renders correctly
- that the search functionality works as expected when a button is clicked
- test that the NASA logo is rendered correctly and with the correct alt tag
- test that your search results get rendered on screen correctly